.vscode/tasks.json

```diff
@@ -0,0 +1,253 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
```

Comment:
Does this comment format work in JSON files? I.e. was this tested after the addition of the comment?

Reply:
Yes that works. json on steroids ;-)